### PR TITLE
Added Git tag and uncommented publish

### PR DESCRIPTION
**Pull request by AVandana-OSI**
### Jira Story
https://osi-cwds.atlassian.net/browse/FIT-284

## Description
Added the final changes to test the Semantic version and update in the docker and update the manifest.

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 
Just the Jenkins file chnage

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -73,7 +78,7 @@ node('intake-slave') {
           }
         }
       }
-    /* // will be added back once the SemVer Works
+    
       stage('Publish') {
         withDockerRegistry([credentialsId: '6ba8d05c-ca13-4818-8329-15d41a089ec0']) {
```
Comment (wmitchell-cambria) [inline Jenkinsfile]: Should this use the docker_credentials_id variable defined above?

File: Jenkinsfile [hunk 2]
```diff
@@ -59,6 +60,10 @@ node('intake-slave') {
         )
       }
 
+      stage('Tag Repo'){
+        tagGithubRepo(VERSION, github_credentials_id)
```
Comment (wmitchell-cambria) [inline Jenkinsfile]: Does this add any prefix to the version when it tags it, or does it just tag `1.2.3` directly?

File: Jenkinsfile (hunk 1 above)
Comment (AVandana-OSI) [reply Jenkinsfile]: That was there before only, I can change it

File: Jenkinsfile (hunk 2 above)
Comment (AVandana-OSI) [reply Jenkinsfile]: Just Tag's like 1.2.3 based on the PR label(MAJOR, MINOR, PATCH)

**Comment by AVandana-OSI** [comment]
retest this please

**Pull request merged**